### DISCUSSION TITLE
I want to add a command into our Listeners, but the program tell me laravel command not found

### DISCUSSION THREAD

**Discussion by xxl4** [laravel/framework#50831] [Q&A]
### Laravel Version

10.41

### PHP Version

8.1.26

### Database Driver & Version

Ubuntu 22.04.4 LTS

### Description

Dear all

`Log::info("send order to shopify list ");
Artisan::call('shopify:order:post', ['--order_id'=> $order->id]);`

Error report on 

![image](https://github.com/laravel/framework/assets/34465153/96420d97-8686-4700-a30b-f04a0bcf958c)

and when i use 

`php artisan list`

![image](https://github.com/laravel/framework/assets/34465153/774c8700-89d1-4187-88d1-5a986e4dd088)

Thank you.

### Steps To Reproduce

none

**Comment by janzcio** [comment]
Hi! Instead of passing an array on your command try to parse it or encode it.

Old:
Artisan::call('shopify:order:post', ['--order_id'=> $order->id]); 

Old:

$parsedArr = json_encode(['--order_id'=> $order->id]);
Artisan::call("shopify:order:post $parsedArr ");
